modules/cache/jvm/src/main/scala/coursier/cache/CacheUrl.scala

```diff
@@ -231,6 +225,7 @@ object CacheUrl {
     credentials: Seq[DirectCredentials] = Nil,
     sslSocketFactoryOpt: Option[SSLSocketFactory] = None,
     hostnameVerifierOpt: Option[HostnameVerifier] = None,
+    authRealmOpt: Option[String] = None,
```

Comment:
Adding this one shouldn't be needed. It's a deprecated method, that shouldn't be used anymore.